api/mongo_config.py

```diff
@@ -6,6 +6,8 @@ def __init__(self):
         self._MONGO_PASSWORD = os.environ.get('MONGO_PASSWORD')
         self._MONGO_CLUSTER_NAME = os.environ.get('MONGO_CLUSTER_NAME')
         self._MONGO_DEFAULT_DB_NAME = os.environ.get('MONGO_DB_NAME')
+        self._SECRET_KEY = os.environ.get('SECRET_KEY')
+        self._LOGIN_DISABLED = os.environ.get('LOGIN_DISABLED', None)
```

Comment:
These properties no longer really relate to mongo, this might be better off just as a `Config`  class or similar